build.gradle

```diff
@@ -61,6 +61,8 @@ dependencies {
     implementation 'com.amazonaws:aws-java-sdk-ec2'
     implementation 'com.amazonaws:aws-java-sdk-iam'
     implementation 'com.amazonaws:aws-java-sdk-cloudwatch'
+    implementation 'com.amazonaws:aws-java-sdk-ecs'
+    implementation 'com.amazonaws:aws-java-sdk-logs'
```

Comment:
what's the purpose of adding log sdk?

Reply:
When creating ecs tasks, I created the log groups for each task first and set it into ContainerDefinition for ECS tasks.

Reply:
ecs should be able to create log group automatically, you can add below configuration into the def.

```
"logConfiguration": {

  "logDriver": "awslogs",
  "options": {
   "awslogs-group": "/ecs/aocintegtest",
  "awslogs-region": "us-west-2",
  "awslogs-stream-prefix": "ecs",
   "awslogs-create-group": "true"
   }
 }
```